website/docs/components/form/segmented-group/index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Segmented Group
+description: Combines one or more input fields and actions to handle complex filtering and data collection.
+caption: Combines one or more input fields and actions to handle complex filtering and data collection.
+status: experimental
+links:
+  figma: https://www.figma.com/file/noyY6dUMDYjmySpHcMjhkN/HDS-Product---Components?node-id=30150%3A45701&t=akpWrhSdwygOH6md-1
+previewImage: assets/illustrations/components/form/segmented-group.jpg
+navigation:
+  keywords: [ 'filter', 'connected', 'fields', 'set' ]
+---
+
+<section data-tab="Guidelines">
+  @include "partials/guidelines/overview.md"
+  @include "partials/guidelines/guidelines.md"
+</section>
+
+<section data-tab="Code">
+</section>
+
+<section data-tab="Specifications">
+  @include "partials/specifications/anatomy.md"
+</section>
+
+<section data-tab="Accessibility">
```

Comment:
the "Accessibility" section is empty

Reply:
I will raise a PR against this one with code docs and accessibility